modules/components/analytics/config/org.wso2.carbon.pc.analytics.config/src/main/java/org/wso2/carbon/pc/analytics/config/clients/LoginAdminServiceClient.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.pc.analytics.config.clients;
+
+/**
+ * Connects with DAS's LoginAdminService
+ */
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.transport.http.HTTPConstants;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.authenticator.stub.AuthenticationAdminStub;
+import org.wso2.carbon.authenticator.stub.LoginAuthenticationExceptionException;
+import org.wso2.carbon.authenticator.stub.LogoutAuthenticationExceptionException;
+import org.apache.axis2.context.ServiceContext;
+
+import java.rmi.RemoteException;
+
+public class LoginAdminServiceClient {
+    private final String serviceName = "AuthenticationAdmin";
+    private AuthenticationAdminStub authenticationAdminStub;
+    private String endPoint;
+    private static final Log log = LogFactory.getLog(LoginAdminServiceClient.class);
+
+    public LoginAdminServiceClient(String backEndUrl) throws AxisFault {
+        this.endPoint = backEndUrl + "/services/" + serviceName;
+        authenticationAdminStub = new AuthenticationAdminStub(endPoint);
+    }
+
+    public String authenticate(String userName, String password)
+            throws RemoteException, LoginAuthenticationExceptionException {
+
+        String sessionCookie = null;
+
+        if (authenticationAdminStub.login(userName, password, "localhost")) {
```

Comment:
Always get server names, ports, etc from a config file. Replace all references to "localhost" with server address retrieved from a config file.